pkg/chart/metadata.go

```diff
@@ -65,3 +67,20 @@ type Metadata struct {
 	// Specifies the chart type: application or library
 	Type string `json:"type,omitempty"`
 }
+
+func (md *Metadata) Validate() error {
+	if md == nil {
+		return errors.New("metadata is required")
+	}
+	if md.APIVersion == "" {
+		return errors.New("metadata apiVersion is required")
+	}
+	if md.Name == "" {
+		return errors.New("metadata name is required")
+	}
+	if md.Version == "" {
+		return errors.New("metadata version is required")
+	}
+	// TODO validate valid semver here?
```

Comment:
did you mean to leave this TODO in here?

Reply:
I did!